config/routes.rb

```diff
@@ -26,6 +26,16 @@
     delete 'logga_ut' => 'devise/sessions#destroy', as: :destroy_user_session
   end
 
+  # tags and exlinks
+  resources :ex_links
+
+  namespace :admin do
+    get 'ex_links/del_unused_tags' => 'ex_links#del_unused_tags'
+    get 'ex_links/check_dead' => 'ex_links#check_dead'
+    get 'ex_links/check_expired' => 'ex_links#check_expired'
+    resources :ex_links
```

Comment:
When I change routes to `get :del_unused_tags, on: :collection` and link to what you proposed, I get:
```
NameError at /admin/ex_links
undefined local variable or method `admin_ex_link_del_unused_tags' for #<#<Class:0x0055c738c5fad0>:0x007f36a2223a78>
Did you mean?  admin_ex_links_url
```
adding `_path` or `_url` doesn't help.